events/tracks/gta2024-ml.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: event-track
+
+title: Mashine Learning
+description: Learn all about 
+
+contributions:
+    organisers:
+        - anuprulez
+    instructors:
+        - anuprulez
```

Comment:
add `- hujambo-dunia` (for Michelle) to instructors. Emmanuel's GitHub handle I am not sure about.